x-pack/legacy/plugins/uptime/public/state/api/monitor_status.ts

```diff
@@ -5,9 +5,13 @@
  */
 
 import { QueryParams } from '../actions/types';
-import { Ping } from '../../../common/graphql/types';
-import { apiService } from './utils';
+import { Ping } from '../../../common/runtime_types';
 import { API_URLS } from '../../../common/constants';
+import { apiService } from './utils';
+
+export interface APIParams {
+  monitorId: string;
+}
```

Comment:
Can we use already defined `MonitorIdParam` in `state/actions/types.ts`

Reply:
I actually don't think that interface was even in use, so I removed it https://github.com/elastic/kibana/pull/59392/commits/b0ca803956f9a2d9114ea5ee659eb94eb107e76f.

Reply:
I've removed this interface in https://github.com/elastic/kibana/pull/59392/commits/b0ca803956f9a2d9114ea5ee659eb94eb107e76f.